### PR TITLE
Implement metasync.every()

### PR DESCRIPTION
**Pull request by DzyubSpirit**
Fixes: #62

### CODE REVIEW

File: lib/array.js [hunk 1]
```diff
@@ -198,4 +198,33 @@ module.exports = (api) => {
...
+    //     fits - true/false returned from fn
+    done // optional on done callback function(err, result)
+    //   err - error or null
+    //   result - true/false, does every element of array fit predicate
+  ) => {
+    done = api.metasync.cb(done);
+
+    let proceedItemsCount = 0;
+
+    function handleFnResult(err, first) {
```
Comment (nechaido) [inline lib/array.js]: @DzyubSpirit I'd prefer `first` to be named `accepted`

File: lib/array.js [hunk 2]
```diff
@@ -198,4 +198,33 @@ module.exports = (api) => {
...
+    //     fits - true/false returned from fn
+    done // optional on done callback function(err, result)
+    //   err - error or null
+    //   result - true/false, does every element of array fit predicate
+  ) => {
+    done = api.metasync.cb(done);
+
+    let proceedItemsCount = 0;
+
+    function handleFnResult(err, accepted) {
```
Comment (tshemsedinov) [inline lib/array.js]: Rename `handleFnResult` to `finish`

File: lib/array.js [hunk 3]
```diff
@@ -198,4 +198,33 @@ module.exports = (api) => {
...
+      if (err) return done(err);
+      if (!accepted) return done(null, false);
+
+      proceedItemsCount++;
+      if (proceedItemsCount === items.length) done(null, true);
+    }
+
+    for (const item of items) {
```
Comment (tshemsedinov) [inline lib/array.js]: Better `let item; for (item in items)`

Comment (DzyubSpirit) [inline lib/array.js]: @tshemsedinov Do you mean `let item; for (item of items)`?
Also maybe it's good case for using `for (var item of items)`? What is difference?

Comment (DzyubSpirit) [inline lib/array.js]: @tshemsedinov ping

**Comment by DzyubSpirit** [comment]
@nechaido Could you please update your review status?

**Comment by DzyubSpirit** [comment]
Landed in c582073

**Pull request closed without merging**